chain/indexer/README.md

```diff
@@ -0,0 +1,91 @@
+# NEAR Indexer
+
+NEAR Indexer is a micro-framework, which provides you with a stream of blocks that are recorded on NEAR network.
+
+
+NEAR Indexer is useful to handle real-time "events" on the chain.
+
+
+NEAR Indexer is going to be used to build NEAR Explorer, augment NEAR Wallet, and provide overview of events in Rainbow Bridge.
+
+
+See the [example](https://github.com/nearprotocol/nearcore/tree/master/tools/indexer/example) for further details.
+
+
+## How to set up and test NEAR Indexer
+
+Assuming you have all necessary tools installed (Python, Rust, Cargo etc.)
+
+### Localnet
+
+Clone [nearcore](https://github.com/nearprotocol/nearcore)
+
+```bash
+$ git clone git@github.com:nearprotocol/nearcore.git
+$ cd nearcore
+$ cargo run --package neard --bin neard init
+```
+
+The above commands should initialize necessary configs and keys to run localnet
+
+```bash
+$ cd tools/indexer/example
+$ cargo run
+```
+
+After that you should see logs of every block produced by your localnet
+
+
+### Betanet
+
+
+Clone [nearcore](https://github.com/nearprotocol/nearcore)
+
+```bash
+$ git clone git@github.com:nearprotocol/nearcore.git
+$ cd nearcore
+$ make release
+```
+
+This will take awhile to build `nearcore` (we are building `release` as it works a bit faster than debug)
+
+To connect to Betanet we need to have a proper configs and keys.
+
+Clone [nearup](https://github.com/near/nearup)
+
+```bash
+$ git clone git@github.com:near/nearup.git
+$ cd nearup
+$ ./nearup betanet --nodocker --binary-path path/to/nearcore/target/release
+```
+
+After that you will be asked for your validator account ID, you can leave it empty as we are not going to validate.
```

Comment:
Just reference nearup and say that one should follow the usual instructions running a non-validating node (leaving the account ID empty). This way we don't need to update our instructions every time the process changes.

BTW, the configs and keys are prepopulated automatically, which is the only reason we use nearup in the first place, so we may as well clearly state that the whole pre-running step is required to get the necessary configs prepared for us.